### PR TITLE
license-headers change to eks module

### PR DESCRIPTION
**Pull request by srinivasreddych**
*Issue #, if available:*

*Description of changes:*
license-headers change to eks module

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: modules/compute/eks/requirements.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ cattrs==22.1.0
     # via jsii
 cdk-nag==2.12.29
     # via -r requirements.in
```
Comment (dgraeber) [inline modules/compute/eks/requirements.txt:36]: Why this change?  I think the cryptography lib was updated by the bots?

Comment (srinivasreddych) [reply modules/compute/eks/requirements.txt]: Can you verify now?

**Review by dgraeber — APPROVED** [review]
